app/sass/_config.scss

```diff
@@ -4,15 +4,38 @@ $color-scheme1: #18ce76; // light green
 $color-scheme2: #2185c5; // sky blue
 $color-scheme3: #3e454c; // dark grey
 $color-scheme4: #767d84; // light grey
+$color-scheme5: #ffffff; // white - page background
+$color-scheme6: #F0F00E; // yellow - shit button
+$color-scheme7: #F00E0E; // red - fuck button
 
 //Typography
 
-#Fira Mono
-https://www.google.com/fonts/specimen/Fira+Mono
+//Fira Mono
+$primary-mono: 'Fira', mono;
 
-#Fira Sans
-https://www.google.com/fonts/specimen/Fira+Sans
+//Fira Sans
+$primary-sans-serif: 'Fira', sans-serif;
 
-#Proxima Nova Soft
-<script src="https://use.typekit.net/epm4awk.js"></script>
-<script>try{Typekit.load({ async: true });}catch(e){}</script>
\ No newline at end of file
+//Proxima Nova Soft
+$heading-san-serif: 'Proxima', sans-serif;
+
+//Heading
+$heading-bg: $color-scheme1;
+
+//Container
+$container: {
```

Comment:
I wouldn't consider lines 26 - 41 config styles. Although they are variables which is what should comprise a config file. Two problems here 1) the formatting is incorrect.

### improper (broken?)
```
$button-ok: {
  @include blah;
}
```

vs.

### Proper
```
$button-ok: @include blah;
```

2) There should be no actual element declarations in the config file.